system/lib/wasmfs/support.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Emscripten Authors.  All rights reserved.
+// Emscripten is available under two separate licenses, the MIT license and the
+// University of Illinois/NCSA Open Source License.  Both these licenses can be
+// found in the LICENSE file.
+
+#include <iostream>
+
+#include "support.h"
+
+namespace wasmfs {
+
+void handle_unreachable(const char* msg, const char* file, unsigned line) {
+#ifndef NDEBUG
+  if (msg) {
+    std::cerr << msg << "\n";
```

Comment:
Should we use `emscripten_console_errorf` here to avoid using any additional file system code?

Reply:
Very good point! Changed to that.